packages/material-ui/src/Accordion/Accordion.test.js

```diff
@@ -26,101 +32,94 @@ describe('<Accordion />', () => {
   }));
 
   it('should render and not be controlled', () => {
-    const wrapper = mount(<Accordion>{minimalChildren}</Accordion>);
-    const root = wrapper.find(`.${classes.root}`).first();
-    expect(root.type()).to.equal(Paper);
-    expect(root.props().square).to.equal(false);
-    expect(root.hasClass(classes.expanded)).to.equal(false);
+    const { container } = render(<Accordion>{minimalChildren}</Accordion>);
+    const root = container.firstChild;
+    expect(root).to.not.have.class(classes.expanded);
   });
 
   it('should handle defaultExpanded prop', () => {
-    const wrapper = mount(<Accordion defaultExpanded>{minimalChildren}</Accordion>);
-    expect(findOutermostIntrinsic(wrapper).hasClass(classes.expanded)).to.equal(true);
+    const { container } = render(<Accordion defaultExpanded>{minimalChildren}</Accordion>);
+    expect(container.firstChild).to.have.class(classes.expanded);
   });
 
   it('should render the summary and collapse elements', () => {
-    const wrapper = mount(
+    const { getByText } = render(
       <Accordion>
         <AccordionSummary>Summary</AccordionSummary>
         <div id="panel-content">Hello</div>
       </Accordion>,
     );
-
-    expect(wrapper.find('[aria-expanded=false]').hostNodes().text()).to.equal('Summary');
-    expect(wrapper.find(Collapse).find('div#panel-content').text()).to.equal('Hello');
+    expect(getByText("Summary")).to.not.have.attribute("aria-expanded");
```

Comment:
This test isn't equivalent to the previous one. It should be something like:
```suggestion
    expect(getByText("Summary")).toBeVisible();
    expect(getByRole('button')).to.have.attribute('aria-expanded','false');
```